### DISCUSSION TITLE
How to configure MIW for Sokrates and Plato connectors ?

### DISCUSSION THREAD

**Discussion by HenkySerpent** [eclipse-tractusx/tractusx-edc#696] [Q&A]
I am following this repo: ([https://github.com/eclipse-tractusx/tractusx-edc/tree/main/docs/samples/example-dataspace](sokrates and plato))

connectors are successfully running, the API for create asset, policy and contract definition all worked fine.
The **API for fetch Catalog is failing**.
From my understanding that API is calling keycloak token service and after that trying to call **API GET /api/credentials of MIW**.
where it is returning 401.

there is no credentials configured. How to do that ? could not able to find theb documentation regarding that. Please provide necessary configuration properties also. 
I have created clients in keycloak for plato and sokrates and was able to call some api to create wallets in MIW.

**Comment by stephanbcbauer** [comment]
@HenkySerpent  thanks for starting the discussion. Is this a redundant discussion to #676  ?

**Reply by HenkySerpent** [reply]
I dont think so. Since I am facing issue in the **fetch catalog API** where #676 didnt get that far I guess.

**Reply by HenkySerpent** [reply]
Will you please give any insights on my issue ? thank you

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
